docs/directory-api-reference/rest.mdx

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_label: REST
+title: Directory Guide
+description: How to use the Aserto Directory REST API
+---
+
+# REST APIs
+You can find documentation for our REST APIs and try them out in our [readme.io hub](https://aserto.readme.io/reference).
+
+# Open API
+You can download our open API specification [here](https://directory.ronen.eng.aserto.com/openapi.json)
```

Comment:
not ready to show this to the world yet!  (it's a sandbox)

Reply:
Removed